### DISCUSSION TITLE
Converting between display and invariant names

### DISCUSSION THREAD

**Discussion by MarkMpn** [microsoft/Power-Fx#2259] [Q&A] [answered]
I would like to be able to convert a formula between the display name representation:

```
First(Accounts).'Account Name'
```

and the invariant version:

```
First(accounts).name
```

I can get to the invariant version using the `CheckResult.ApplyGetInvariant` method - how should I do the conversion back again when I want to load the formula for display?

**Comment by MikeStall** [comment, marked as answer]
There are helpers on the Engine class.
See Engine.GetDisplayExpression(): 

https://github.com/microsoft/Power-Fx/blob/bf5c43807548b1f78caad918f827257539598487/src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs#L506
